tests/nested/manual/muinstaller-core/task.yaml

```diff
@@ -93,6 +93,9 @@ execute: |
   # make the seed label more predictable for muinstaller auto-mode
   LABEL=core
   mv ./"$SEED_DIR"/system-seed/systems/* ./"$SEED_DIR"/system-seed/systems/"$LABEL"
+  # since we change the label, make sure that the path to the kernel in grubenv
+  # is updated
+  sed -i 's|snapd_recovery_kernel=/systems/.*/snaps|snapd_recovery_kernel=/systems/'"${LABEL}"'/snaps|' "./${SEED_DIR}/system-seed/systems/${LABEL}/grubenv"
```

Comment:
Maybe we could simply avoid renaming the label by finding out how is the folder named in `$SEED_DIR`. Iirc setting a fixed label is a left-over of when we used "auto" mode for muinstaller, but we are actually calling it explicitly below.

Reply:
Okay, that sounds simpler. Thanks!